From c5ddc7d6da9733ee471b4287092cb05108a0fb73 Mon Sep 17 00:00:00 2001
From: Oliver Beard <olib141@outlook.com>
Date: Mon, 12 Jan 2026 19:30:51 +0000
Subject: [PATCH] wip

---
 src/frontend/kcm/kcm.cpp | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/src/frontend/kcm/kcm.cpp b/src/frontend/kcm/kcm.cpp
index 2edbd395..92f926b4 100644
--- a/src/frontend/kcm/kcm.cpp
+++ b/src/frontend/kcm/kcm.cpp
@@ -94,6 +94,12 @@ void PlasmaLoginKcm::save()
     const QString wallpaperPluginId = PlasmaLoginSettings::getInstance().wallpaperPluginId();
     for (const QString &wallpaperKey : wallpaperConfiguration()->keys()) {
         const QVariant value = wallpaperConfiguration()->value(wallpaperKey);
+        if (value.toString().isEmpty()) {
+            // Writing out an empty (default) value is going to
+            // override any configuration-specified defaults
+            continue;
+        }
+
         tempConfig.group(QLatin1String("Greeter"))
             .group(QLatin1String("Wallpaper"))
             .group(wallpaperPluginId)
-- 
GitLab

